pkg/kubelet/kubelet_pods_test.go

```diff
@@ -201,39 +201,49 @@ func writeHostsFile(filename string, cfg string) (string, error) {
 
 func TestManagedHostsFileContent(t *testing.T) {
 	testCases := []struct {
-		hostIPs         []string
-		hostName        string
-		hostDomainName  string
-		hostAliases     []v1.HostAlias
-		expectedContent string
+		hostIPs                       []string
+		hostName                      string
+		hostDomainName                string
+		hostAliases                   []v1.HostAlias
+		isSupportIPv6                 bool
+		expectedContentSupportIPv6    string
+		expectedContentNotSupportIPv6 string
```

Comment:
You have both `isSupportIPv6` (which you don't use) and ipv6-vs-non-ipv6 `expected` here. I think it would be simpler to just do the `isSupportIPv6` thing, and just have it set in some tests and not in others.